bitnami/postgresql-ha/Chart.yaml

```
@@ -40,4 +40,4 @@ maintainers:
 name: postgresql-ha
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/postgresql-ha
-version: 12.1.3
+version: 12.1.4
```

I think is better to use a minor update because we are adding a new feature.
```suggestion
version: 12.2.0
```